API.md

```diff
@@ -0,0 +1,664 @@
+}
+```
+
+## Get Books
+```
+GET /books
+```
+
+**Response**
+```
+Status 200
+  
+[
+  {
+    "id": 33,
+    "title": "Sprint : How to Solve Big Problems and Test New Ideas in Just 5 Days",
+    "author": "Jake Knapp y John Zeratsky",
+    "description": "The sprint is a five-day process for answering critical business questions through design, prototyping, and testing ideas with customers.",
+    "image": "https://images-na.ssl-images-amazon.com/images/I/512V%2BzNxQ9L._SX327_BO1,204,203,200_.jpg"
+  },
+  ...
+]
+```
+
+## Order a Book
+```
+POST /books/order
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| :---: |:---:| ---|
+| date | String | Date |
+| patient | String | Patient username |
+| referenceNumber | number | Book ID |
+
+**Example**
+```
+{
+  "date": "2017-5-1T14:00:00Z",
+  "patient": "john",
+  "referenceNumber": 33
+}
+```
+
+**Response**
+```
+Status 201
+  
+{
+  "id": 600,
+  "date": "2017-5-1T14:00:00Z",
+  "patient": "john",
+  "referenceNumber": 33
+}
+```
+
+# Meals
+## Get Next Menu
+```
+GET /menus/next
+```
+
+**Response**
+```
+Status 200
+  
+{
+  "date": "2017-5-1T14:00:00Z",
+  "firstCourses": [
+    {
+      "name": "Salad",
+      "description": "Tomatoes, lettice and bacon.",
+      "image": "https://www.google.es/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjknLawgs3TAhULVxQKHfbcAvMQjRwIBw&url=http%3A%2F%2Fjetspizza.com%2Fmenu%2Fsalads&psig=AFQjCNE447cnpJfGRhiwiN5Hys_XLrgpmQ&ust=1493670502918204"
+    },
+    ...
+  ],
+  "secondCourses": [
+    {
+      "name": "Salad",
+      "description": "Tomatoes, lettice and bacon.",
+      "image": "https://www.google.es/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjknLawgs3TAhULVxQKHfbcAvMQjRwIBw&url=http%3A%2F%2Fjetspizza.com%2Fmenu%2Fsalads&psig=AFQjCNE447cnpJfGRhiwiN5Hys_XLrgpmQ&ust=1493670502918204"
+    },
+    ...
+  ],
+  "desserts": [
+   {
+      "name": "Salad",
+      "description": "Tomatoes, lettice and bacon."
+      "image": "https://www.google.es/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjknLawgs3TAhULVxQKHfbcAvMQjRwIBw&url=http%3A%2F%2Fjetspizza.com%2Fmenu%2Fsalads&psig=AFQjCNE447cnpJfGRhiwiN5Hys_XLrgpmQ&ust=1493670502918204"
+    },
+    ...
+  ]
+}
+```
+
+## Menu choice
+```
+POST /menus/order
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| :---: |:---:| ---|
+| date | String | Date |
+| patient | String | Patient username |
+| firstCourse | number | Dish ID |
+| secondCourse | number | Dish ID |
+| dessert | number | Dish ID |
+
+**Example**
+```
+{
+  "date": "2017-5-1T14:00:00Z",
+  "patient": "john",
+  "firstCourse": 33,
+  "secondCourse": 43,
+  "dessert": 53
+}
+```
+
+**Response**
+```
+Status 201
+  
+{
+  "id": 33,
+  "date": "2017-5-1T14:00:00Z",
+  "patient": "john",
+  "firstCourse": 33,
+  "secondCourse": 43,
+  "dessert": 53
+}
+```
+
+
+# Items
+## Add item
```

Comment:
For the Shop Epic: add, remove and edit items are not required methods. We are not going to implement that part in workers app.